cfme/v2v/migrations.py

```diff
@@ -166,16 +167,17 @@ def fill(self, values):
                         ...
                     }
         """
-        for cluster in values:
-            if self.cluster_selector.is_displayed:
-                self.cluster_selector.fill(cluster)
-            for mapping in values[cluster]['mappings']:
-                self.source_datastores.fill(mapping['sources'])
-                self.target_datastores.fill(mapping['target'])
-                self.add_mapping.click()
+        if values:
+            for cluster in values:
+                if self.cluster_selector.is_displayed:
+                    self.cluster_selector.fill(cluster)
+                for mapping in values[cluster]['mappings']:
+                    self.source_datastores.fill(mapping['sources'])
+                    self.target_datastores.fill(mapping['target'])
+                    self.add_mapping.click()
         was_change = not self.mappings_tree.is_empty
-        if was_change:
-            self.logger.info("Fill operation was successful, click Next.")
+        self.cancel_btn.wait_displayed()
```

Comment:
Same comment @kedark3 made above about removing `if was_change`

Reply:
will see the result of this in PRT and then run other tests too